src/libostree/ostree-repo-pull.c

```diff
@@ -4903,6 +4905,7 @@ find_remotes_cb (GObject      *obj,
   g_autoptr(OstreeFetcher) fetcher = NULL;
   g_autofree const gchar **ref_to_latest_commit = NULL;  /* indexed as @refs; (element-type commit-checksum) */
   gsize n_refs;
+  g_autofree char **override_commit_ids = NULL;
```

Comment:
I think this should be `const`: `g_autofree const char **override_commit_ids = NULL`.

Reply:
Hmmm...that's not the style we use elsewhere though.  I am going back and forth on whether or not it's "technically" right.

This is (somewhat) personal opinion but I generally don't use `const` in C except for `const char *` since [as Wikipedia says](https://en.wikipedia.org/wiki/Const_(computer_programming)) those are stored in read-only memory often and to me it's really more about "owned" vs "borrowed" in Rust terms but...

Some parts of glib use `const char * const *` which is more technically right but also annoying since other things use `char**` and so in the end one just needs to cast, and really we're buried in legacy here and why doesn't someone invent a time machine and go back and give us Rust 10 years ago...